arangod/Cluster/ClusterFeature.cpp

```diff
@@ -680,18 +680,6 @@ void ClusterFeature::prepare() {
   }
 
   // This must remain here for proper function after hot restores
```

Comment:
The original code comment ("this must remain here...") is now obsolete. I am wondering if the comment was actually correct, now that you moved the agency cache startup to somewhere else.

Reply:
Discussed with @kvahed - seems to be outdated - removed!